doc/source/user_guide/ug_case_settings.rst

```diff
@@ -356,7 +356,7 @@ dynamics_nml
 
    "", "", "", ""
    "``advection``", "``remap``", "linear remapping advection scheme", "``remap``"
-   "", "``none``", "advection off", ""
```

Comment:
need to check whether advection=none works correctly in the reverted code

Reply:
I tried "none" and it worked, but then noticed that the advection timer was still pretty large.  The issue is that in step_dyn_horiz, we have

```
      if (advection == 'upwind') then
         call transport_upwind (dt)    ! upwind                                                                                                                     
      else
         call transport_remap (dt)     ! incremental remapping                                                                                                      
      endif
```

which is a bit of a problem.  Unless you set 'upwind', the advection option will be remap.  In ice_init, remap, upwind, and none are the only accepted options.  So, either we need to remove none as an option or we need to update the code.  Is it as simple as

```
      if (advection == 'upwind') then
         call transport_upwind (dt)    ! upwind                                                                                                                     
      elseif (advection == 'remap') then
         call transport_remap (dt)     ! incremental remapping                                                                                                      
      endif
```

We might as well fix this now while we're thinking about it.  How do you want me to proceed?

Reply:
I like this:

> elseif (advection == 'remap') then

Reply:
We also have ktransport, which is supposed to turn it off when it is -1.  is it not wrapped around that 'if advection' code block?

Reply:
Happy to have either/both.  In any case, we either need to remove "none" as a supported advection option or switch to the elseif assuming there are no other issues.  I will test the elseif and you can think about it.

Reply:
Just to clarify, the entire if block is

```
      if (ktransport > 0) then
      if (advection == 'upwind') then
         call transport_upwind (dt)    ! upwind                                                 
      elseif (advection == 'remap') then
         call transport_remap (dt)     ! incremental remapping                                  
      endif
      endif

The only other place ktransport seems to show up is in ice_init.  It seems like we can get rid of ktransport completely and rely only on advection?  We can leave it in the namelist for backwards compatibility.  Maybe ktransport<0 should set advection to 'none' in ice_init (overriding which we try to avoid in general, but it seem required here) and then we get rid of ktransport if check above?
```
so maybe we don't need to support both?

Reply:
Adding the else if seems to have turned advection off with advection=none.  And it completed with Advection timer = 0.00.   So no advection does seem to work for an out of the box case.

The only test we have right now with advection off is "box2001" which sets ktransport=-1.  Do we need a test with advection='none' with the gx3 grid?

Reply:
I don't think we need a test with advection = 'none'.  We didn't used to have ktransport, but if I remember correctly, @TillRasmussen added it to make the on/off flags more consistent, and we kept 'advection' for backwards compatibility.  Maybe we don't need 'none', but it helps clarify the namelist when ktransport=-1 or 0.  I still like the elseif change, so that remap isn't the default when ktransport > 0.  Although I'm not imagining a situation where we'd have ktransport>0 and advection='none'.  I agree with you, we either need to get rid of the 'none' option for advection or else have this longer elseif statement.  I'm fine either way.

Reply:
If you leave it up to me, I'll deprecate ktransport except as a namelist and have ktransport=-1 override the advection setting, but otherwise have 'upwind', 'remap', and 'none' be working advection options.  I will implement this and run the test suite. @TillRasmussen or anyone, is there a reason not to do that?

Reply:
Sounds fine to me.  ktransport was added in #151 and I was wrong about who did it -- @mattdturner not @TillRasmussen (sorry Till)

Reply:
The lastest commit, 6aeb366, implements the new proposed handling of advection='none' and deprecation of ktransport.